### DISCUSSION TITLE
Is there a way to optionally disable the tray icon?

### DISCUSSION THREAD

**Discussion by jonnyborbs** [johnste/finicky#254] [General]
Love that Finicky is so transparent and can just set-and-forget. I'd love it even more if I could disable the menu bar icon when the app is running. Is there a config handle for that, or would it be core product functionality that's just lacking?

Thanks!

**Comment by johnste** [comment]
`options.hideIcon` is your friend, as noted in the [wiki](https://github.com/johnste/finicky/wiki/Configuration)

```
module.exports = {
  defaultBrowser: "Google Chrome",
  options: {
    // Hide the finicky icon from the top bar. Default: false
    hideIcon: false, 
  },
};
```

**Reply by jonnyborbs** [reply]
Hey, thanks! I don't know how I missed that. I was looking through the config options yesterday and must have scanned right past this. Appreciate the note.

**Reply by johnste** [reply]
You're welcome, I hope you enjoy using Finicky :)
